src/config-manager.ts

```diff
@@ -88,15 +88,15 @@ if (!config.http.port) {
 }
 
 if (md5(config.crypto.wup.aes_key) !== BOSS_WIIU_AES_KEY_MD5_HASH) {
-	errors.push('Invalid BOSS WiiU AES key. Set or correct the PN_BOSS_CONFIG_BOSS_WIIU_AES_KEY environment variable');
+	warnings.push('Invalid BOSS WiiU AES key. Uploading and encrypting new BOSS content for the Wii U won\'t work! Set or correct the PN_BOSS_CONFIG_BOSS_WIIU_AES_KEY environment variable');
 }
 
 if (md5(config.crypto.wup.hmac_key) !== BOSS_WIIU_HMAC_KEY_MD5_HASH) {
-	errors.push('Invalid BOSS WiiU HMAC key. Set or correct the PN_BOSS_CONFIG_BOSS_WIIU_HMAC_KEY environment variable');
+	warnings.push('Invalid BOSS WiiU HMAC key. Uploading and encrypting new BOSS content for the Wii U won\'t work! Set or correct the PN_BOSS_CONFIG_BOSS_WIIU_HMAC_KEY environment variable');
 }
 
 if (md5(config.crypto.ctr.aes_key) !== BOSS_3DS_AES_KEY_MD5_HASH) {
-	errors.push('Invalid BOSS 3DS AES key. Set or correct the PN_BOSS_CONFIG_BOSS_3DS_AES_KEY environment variable');
+	warnings.push('Invalid BOSS 3DS AES key. Uploading and encrypting new BOSS content for the 3DS won\'t work! Set or correct the PN_BOSS_CONFIG_BOSS_3DS_AES_KEY environment variable');
```

Comment:
This is a pretty critical error, is there a reason why this was changed? Eventually these keys will be 100% required, since there are some games which use dynamic/"on demand" BOSS data (PIKMIN 3 for example downloads some score related data from BOSS, which obviously will update fairly frequently and thus will likely just be generated on demand)

Reply:
The issue is that self-hosters might only own a Wii U or a 3DS (not both). These people are excluded from using the entire BOSS server if both console's keys are required. I believe it is better to at least allow the server to start up if the keys aren't provided so that users aren't required to own both consoles.

Adding dynamic BOSS data shouldn't be a problem here. If the user doesn't own a Wii U to provide the keys, the dynamic BOSS data generation for Wii U data shouldn't be necessary anyway.

Reply:
I disagree with your last point, but you are right that currently someone must own BOTH consoles. I had not consider this, that's a very fair point. Sounds reasonable to me :+1: